sdk/rust/Cargo.toml

```diff
@@ -3,8 +3,17 @@ name = "spin-sdk"
 version = { workspace = true }
 authors = { workspace = true }
 edition = { workspace = true }
+license = { workspace = true }
+homepage = { workspace = true }
+repository = { workspace = true }
+documentation = "https://fermyon.github.io/rust-docs/spin/main/spin_sdk"
+readme = "README.md"
 rust-version = "1.64"
```

Comment:
I know this was already here, but just want to note: Are we use this is the correct minimum Rust version? We should probably test any MSRV in CI.

Reply:
Right.  Judging by https://github.com/fermyon/spin/blob/main/Cargo.toml#L7 and https://github.com/fermyon/spin/blob/main/.github/workflows/release.yml#L14, this may need to bump to 1.71.  We could assign it `{ workspace = true }` to inherit the top-level value.  Or, leave alone for now and invest cycles in determining the actual minimum version for this crate.

Reply:
@vdice I think it's safe to pull this from the root `Cargo.toml` like you suggest.

Reply:
Thanks @fibonacci1729; updated in https://github.com/fermyon/spin/pull/2151/commits/1354c4376708f417fdb092f5e7c287036c675738